//------------------------------------------------------------------------------
// File: Devices.proto
// Author: Andreas-Joachim Peters - CERN
//------------------------------------------------------------------------------

syntax="proto3";
package eos.console;

message DevicesProto {

  message LsProto {
    enum OutFormat {
      NONE                  = 0; //
      MONITORING            = 1; // [-m]
      LISTING               = 2; // [-l]
    }
    OutFormat outformat   = 2; //
    bool refresh          = 3; //
  }

oneof subcmd {
    LsProto ls = 1;
  }

}
